src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -646,53 +676,72 @@ public Type DfsTraverse(Node n)
                     EmitGroupExpressionNode(node);
                     break;
             }
+            if(compilePass == CompilePass.GatherTypeInfo)
+            {
+                if (aSTTypeInfoMap.ContainsKey(node.ID))
```

Comment:
Can we rename this to follow camelcase: `astTypeInfomap`, or simply rename to `typeInfoMap`?